react/components/EditorProvider.tsx

```diff
@@ -332,6 +353,15 @@ class EditorProvider extends Component<Props, State> {
       iframeWindow,
       messages,
       mode,
+      onChangeIframeUrl: url => {
```

Comment:
⛏ For uniformity sake, I think it would be better to define it as a class method or regular function.